doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1132 @@
+  * :ref:`lib_azure_iot_hub` library - Improved the internal connection state handling.
+  * :ref:`lte_lc_readme` library - Added support for manufacturer-specific default eDRX/PSM values.
+  * :ref:`liblwm2m_carrier_readme` library - Updated to v0.10.2. See :ref:`liblwm2m_carrier_changelog` for details.
+
+DFU Target
+----------
+
+* Added:
+
+  * New target ``dfu_target_full_modem``, which is used for full modem firmware updates. This requires an external flash memory of a minimum of 2MB to work.
+
+* Updated:
+
+  * Renamed ``dfu_target_modem`` to ``dfu_target_modem_delta``.
+  * Moved the ``dfu_target`` code from :file:`subsys/dfu` to :file:`subsys/dfu/dfu_target`.
+  * Extracted the stream flash memory functionality from ``dfu_target_mcuboot`` to ``dfu_target_stream_flash`` to facilitate code reuse for other ``dfu_targets``, which write large objects to the flash memory.
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+nRF52832 SoC
+------------
+
+* Removed:
+
+  * ``esb`` subystem - Removed support for older SoC revision workarounds.
+
+    * Removed support for nRF52832 revision 1 workarounds for Errata 102, Errata 106, and Errata 107.
+    * Removed support for nRF52832 revision 2 workarounds for Errata 143.
+
+nRF5340 SoC
+-----------
+
+* Added:
+
+  * ``esb`` subsystem - Added support for nRF5340 (CPUNET) in the ESB subsystem.
+  * ``spm`` subsystem - Added support for nRF5340 peripherals in non-secure applications.
+  * :ref:`multiprotocol-rpmsg-sample` sample for Network Core, which enables support for multiprotocol, IEEE 802.15.4, and |BLE| applications.
+
+* Updated:
+
+  * :ref:`ble_samples` -  	Added configuration overlays for child image to the required |BLE| samples so that no Kconfig updates in the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample are needed by default.
+  * :ref:`nrf5340_empty_app_core` sample - Disabled the kernel memory pool option :option:`CONFIG_KERNEL_MEM_POOL` to reduce the memory footprint.
+  * ``bl_boot`` library - Disabled clock interrupts before booting the application. This change fixes an issue where the :ref:`bootloader` sample would not be able to boot a Zephyr application on the nRF5340 SoC.
+
+
+Front-end module (FEM)
+----------------------
+
+* Added support for nRF21540 revision 1 or older to :ref:`direct_test_mode` and :ref:`radio_test` samples.
+* Added support for RF front-end Modules (FEM) in :ref:`mpsl` library. The front-end module feature in MPSL currently supports the SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+
+
+|BLE|
+-----
+
+* Added:
+
+  * Support for nRF21540 front-end module (revision 1 or older) to :ref:`direct_test_mode` sample.
+  * Minimal configuration (:file:`prj_minimal.conf`) to the :ref:`peripheral_uart` sample, thus enabling support for building nRF52810 and nRF52811 devices.
+
```

Comment:
Are we missing cts_client (sample and library) here?
https://github.com/nrfconnect/sdk-nrf/pull/3071

@joerchan

Reply:
Added the sample and library in the changelog section.